manifests/logind.pp

```diff
@@ -0,0 +1,69 @@
+class systemd::logind(
+                        $handle_hibernate_key            = 'hibernate',
+                        $handle_lid_switch               = 'suspend',
+                        $handle_lid_switch_docked        = 'ignore',
+                        $handle_power_key                = 'poweroff',
+                        $handle_suspend_key              = 'suspend',
+                        $hibernate_key_ignore_inhibited  = false,
+                        $holdoff_timeout_sec             = 30,
+                        $idle_action                     = 'ignore',
+                        $idle_action_sec                 = '30min',
+                        $inhibit_delay_max_sec           = 5,
+                        $inhibitors_max                  = 8192,
+                        $kill_exclude_users              = ['root'],
+                        $kill_only_users                 = [],
+                        $kill_user_processes             = true,
+                        $lid_switch_ignore_inhibited     = true,
+                        $n_auto_vts                      = 6,
+                        $power_key_ignore_inhibited      = false,
+                        $remove_ipc                      = yesno2bool($systemd::removeipc),
```

Comment:
It appears as though you may be able to use `str2bool` in `stdlib` for this to introduce fewer dependencies. 
https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/lib/puppet/parser/functions/str2bool.rb#L32